config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  namespace :api do
+    namespace :v1 do
+      resources :greetings, only: [:index]
+    end
+  end
+
+  root 'root#index'
+end
```

Comment:
- [ ] Kindly fix this error that comes up when the user refreshes the screen in order to get a second random greeting
![image](https://user-images.githubusercontent.com/81932478/205466574-a440a557-8c71-465d-8529-3c5712e7bbc9.png)